tsconfig.json

```diff
@@ -24,7 +25,7 @@
   },
   "include": [
     "src/**/*",
```

Comment:
Should we also update this?

Reply:
#TIL 
> Importantly, rootDir does not affect which files become part of the compilation. It has no interaction with the include, exclude, or files tsconfig.json settings.
https://www.typescriptlang.org/tsconfig#rootDir

Reply:
That is confusing, so `includes` only determines which definitions get built?

Reply:
rootDir is used to keep the output's directory structure simple and clean 
includes determines which file (must be in rootDir) will be compiled by tsc

We should have a separate tsconfig.test.json which is only used for ts-jest
In this way ```jest-setup.ts``` will not affect the output's directory structure anymore